README.md

```diff
@@ -1,17 +1,18 @@
-# amazon-q-connectjs
+#amazon-q-connectjs
 
 Amazon Q in Connect, an LLM-enhanced evolution of Amazon Connect Wisdom. The Amazon Q Connect JavaScript library (QConnectJS) gives you the power to build your own Amazon Q Connect widget.
 
 The library uses an Amazon Connect authentication token to make API calls to Amazon Q Connect and supports all Amazon Q Connect `Agent Assistant` functionality. For example, you can manually query knowledge documents, get knowledge content, or start generating automated suggestions.
 
 QConnectJS supports the following APIs:
-* [QueryAssistant](#QueryAssistant)
-* [GetContact](#GetContact)
-* [GetRecommendations](#GetRecommendations)
-* [NotifyRecommendationsReceived](#NotifyRecommendationsReceived)
-* [GetContent](#GetContent)
-* [SearchSessions](#SearchSessions)
-* [ListIntegrationAssociations](#ListIntegrationAssociations)
+* [QueryAssistant](#queryassistant)
```

Comment:
does the case change here matter?

Reply:
case does matter, need to be lower case so the anchor link can work